include/TPP/Passes.td

```diff
@@ -54,6 +54,23 @@ def VectorizationPass : Pass<"vectorization-pass",
 }
 
 
+
+def LinalgTiling : Pass<"tile-linalg"> {
+  let summary = "Tile matmul reduction dimension.";
+  let description = [{
+    Tiles matmul reduction dimension using serial loop.
+  }];
+  let dependentDialects = ["linalg::LinalgDialect",
+                           "memref::MemRefDialect",
+                           "arith::ArithDialect"];
+  let options = [
+         ListOption<"mTileShape", "mTile", "unsigned", "Tile shape for m dim">,
+        ListOption<"nTileShape", "nTile", "unsigned", "Tile shape for n dim">,
```

Comment:
nit: no K-dim tile option?

Reply:
We invoke the pass using option: --tile-linalg="mTile=4,8 nTile=8,16" (for example). So, here the k-dim tile is 8.
The pass will infer the K-dim tile value from the mTile and nTile input. Hence, not added the k-dim option.